tgui/packages/tgui/interfaces/OverwatchConsole.js

```diff
@@ -0,0 +1,809 @@
+                  ))}
+              </Table>
+            </Stack.Item>
+          </Section>
+        </Stack>
+      </Section>
+      <Section>
+        <Table>
+          <Table.Row style={{ 'border-bottom': '1px solid #2F3037' }}>
+            <Table.Cell>Squad Leader</Table.Cell>
+            <Table.Cell>RTO</Table.Cell>
+            <Table.Cell>Specialist</Table.Cell>
+            <Table.Cell>Smartgunner</Table.Cell>
+            <Table.Cell>Medic</Table.Cell>
+            <Table.Cell>Combat Technician</Table.Cell>
+            <Table.Cell>Alive</Table.Cell>
+            <Table.Cell>Deployed</Table.Cell>
+          </Table.Row>
+          <Table.Row>
+            <Table.Cell>
+              <ColorBox
+                color={squad_data.current_squad_leader_name ? 'green' : 'red'}
+              />
+              {squad_data.current_squad_leader_name ? ' Deployed' : ' None'}
+            </Table.Cell>
+            <Table.Cell>
+              <ColorBox color={squad_data.rto > 0 ? 'green' : 'red'} />
+              {squad_data.rto ? ' ' + squad_data.rto + ' Deployed' : ' None'}
+            </Table.Cell>
+            <Table.Cell>
+              <ColorBox color={squad_data.specialists > 0 ? 'green' : 'red'} />
+              {squad_data.specialists ? ' Deployed' : ' None'}
+            </Table.Cell>
+            <Table.Cell>
+              <ColorBox color={squad_data.smartgun > 0 ? 'green' : 'red'} />
+              {squad_data.smartgun
+                ? ' ' + squad_data.smartgun + ' Deployed'
+                : ' None'}
+            </Table.Cell>
+            <Table.Cell>
+              <ColorBox color={squad_data.medics > 0 ? 'green' : 'red'} />
+              {squad_data.medics
+                ? ' ' + squad_data.medics + ' Deployed'
+                : ' None'}
+            </Table.Cell>
+            <Table.Cell>
+              <ColorBox color={squad_data.engineers > 0 ? 'green' : 'red'} />
+              {squad_data.engineers
+                ? ' ' + squad_data.engineers + ' Deployed'
+                : ' None'}
+            </Table.Cell>
+            <Table.Cell>{squad_data.alive}</Table.Cell>
+            <Table.Cell>{squad_data.total_deployed}</Table.Cell>
+          </Table.Row>
+        </Table>
+      </Section>
+    </>
+  );
+};
+
+export const OverwatchSupplies = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { x_supply, y_supply, supply_cooldown, supply_ready, world_time }
+    = data;
+
+  const cooldown = round(supply_cooldown - world_time / 10);
+
+  const [target_x, setTargetX] = useLocalState(context, 'target_x', x_supply);
+  const [target_y, setTargetY] = useLocalState(context, 'target_y', y_supply);
+
+  return (
+    <Section title="Supply Drop">
+      <OverwatchAuthenticated />
+      <ProgressBar
+        ranges={{
+          good: [-Infinity, 200],
+          average: [200, 400],
+          bad: [Infinity, 400],
+        }}
+        minValue={0}
+        maxValue={500}
```

Comment:
this should not be hard coded as a number in the ui and should be got via static data instead because it is liable to change in the codebase